docs/run/primitives.mdx

```diff
@@ -90,6 +90,91 @@ Example:
 estimator.run([(circuit1, observable1, param_values1),(circuit2, observable2, param_values2) ])
 ```
 
+<span id="broadcast-rules"></span>
+##### Broadcasting rules
+
+Estimator V2 aggregates elements from multiple arrays (observables and parameter values) by following the same broadcasting rules as NumPy. This section summarizes those rules.  For a detailed explanation, see the [NumPy broadcasting rules documentation.](https://numpy.org/doc/stable/user/basics.broadcasting.html)
+
+Rules:
+
+* Input arrays do not need to have the same number of dimensions. 
+  * The resulting array will have the same number of dimensions as the input array with the largest dimension. 
+  * The size of each dimension is the largest size of the corresponding dimension.
+  * Missing dimensions are assumed to have size one.
+* Shape comparisons start with the rightmost dimension and continue to the left.
+* Two dimensions are compatible if their sizes are equal or if one of them is 1.
+
+Examples of array pairs that broadcast:
+
+```text
+A1     (1d array):      1
+A2     (2d array):  3 x 5
+Result (2d array):  3 x 5
+
+
+A1     (3d array):  11 x 2 x 7
+A2     (3d array):  11 x 1 x 7
+Result (3d array):  11 x 2 x 7
+```
+
+Examples of array pairs that do not broadcast:
+
+```text
+A1     (1d array):  5
+A2     (1d array):  3 
+
+A1     (2d array):      2 x 1
+A2     (3d array):  6 x 5 x 4 #This would work except for the middle dimensions (2 compared to 5).
```

Comment:
Could this be easier to understand?

```suggestion
A2     (3d array):  6 x 5 x 4 # This would work if the middle dimension were 2, but it is 5.
```